app/helpers/users_helper.rb

```diff
@@ -0,0 +1,5 @@
+module UsersHelper
+  def user_params
+    params.require(:user).permit(:username, :email, :password)
+  end
```

Comment:
- [ ] Please move this into the users' controller and make it a private method, because it deals with the params which shouldn't be public.